normandy/recipes/tests/__init__.py

```diff
@@ -53,6 +71,33 @@ def signed(self, create, extracted=False, **kwargs):
         else:
             return None
 
+    @factory.post_generation
```

Comment:
Someday we'll factor this pattern out to a helper function. But that might not be today.